x-pack/test_serverless/functional/test_suites/common/management/index_management/indices.ts

```diff
@@ -33,14 +35,42 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       await pageObjects.indexManagement.clickCreateIndexSaveButton();
       await pageObjects.indexManagement.expectIndexToExist(testIndexName);
     });
-    it('can manage index', async () => {
-      await pageObjects.indexManagement.selectIndex(testIndexName);
-      await pageObjects.indexManagement.clickManageButton();
-      await pageObjects.indexManagement.contextMenuIsVisible();
-    });
-    it('can delete index', async () => {
-      await pageObjects.indexManagement.confirmDeleteModalIsVisible();
-      await pageObjects.indexManagement.expectIndexIsDeleted(testIndexName);
+
+    describe('manage index', function () {
+      after(async () => {
+        await esDeleteAllIndices(testIndexName);
+      });
+      beforeEach(async () => {
+        await pageObjects.common.navigateToApp('indexManagement');
+        // Navigate to the indices tab
+        await pageObjects.indexManagement.changeTabs('indicesTab');
+        await pageObjects.header.waitUntilLoadingHasFinished();
+        await pageObjects.indexManagement.manageIndex(testIndexName);
+        await pageObjects.indexManagement.manageIndexContextMenuExists();
+      });
+      describe('navigate to index detail tabs', function () {
+        this.tags('skipSvlSearch');
```

Comment:
Now that `search_indices` is not behind feature flag, we don't this test for search solution but for other solution only